jupyterhub_carina/CarinaAuthenticator.py

```diff
@@ -13,7 +13,7 @@ class CarinaLoginHandler(OAuthLoginHandler, OAuth2Mixin):
     _OAUTH_AUTHORIZE_URL = CarinaOAuthClient.CARINA_AUTHORIZE_URL
     _OAUTH_ACCESS_TOKEN_URL = CarinaOAuthClient.CARINA_TOKEN_URL
 
-    scope = ['identity', 'cluster_credentials', 'create_cluster']
+    scope = ['identity', 'read', 'write', 'execute']
```

Comment:
https://getcarina.com/docs/reference/oauth-integration/#interact-with-carina

* Read = read-only operations, such as listing clusters
* Write = create/update/delete clusters
* Execute = Download cluster credentials, i.e. being able to execute code on your cluster